tulflow/harvest.py

```diff
@@ -137,6 +137,7 @@ def perform_xml_lookup(oai_record, **kwargs):
         """Parse additions/updates & add boundwiths."""
 
         if len(cache) == 0:
+            print("*** Fetching CSV lookup file from s3 ***")
```

Comment:
I'd prefer `logging.info("Message")` instead of print

Reply:
OK, thanks.